src/api/app/controllers/webui/package_controller.rb

```diff
@@ -1002,7 +1002,7 @@ def save_meta
         errors << "Package doesn't exists in that project."
       end
 
-    rescue Suse::ValidationError => e
+    rescue Suse::ValidationError, ActiveXML::Transport::Error => e
```

Comment:
What can produce an `ActiveXML::Transport::Error` here? `Suse::Validator.validate`?

Reply:
"Not sure" is not good enough, sorry...